tests/unit/lms/services/group_info_test.py

```diff
@@ -136,6 +127,10 @@ def with_existing_group_infos(self):
         # Add some "noise" GroupInfo to make the tests more realistic
         factories.GroupInfo.build_batch(3)
 
+    @pytest.fixture
+    def lti_params(self, pyramid_request):
```

Comment:
I made the code more explicit in `lms/resources/_js_config/__init__.py` not relying on `.columns()` but I'm not doing the same here. `lti_params` contains the right keys.

Ideally I'd like to basically remove this whole service, note that it doesn't have any read/get methods at the moment. It's only used by the JUdel analytics and I'll have to double check it's usage on the new analytics queries.